classytags/helpers.py

```diff
@@ -83,7 +84,10 @@ def render_tag(self, context, **kwargs):
             safe_context.update(**data)
             output = render_to_string(template, safe_context)
         else:
-            data = context.new(self.get_context(context, **kwargs))
+            new_context = context.new(
+                flatten_context(self.get_context(context, **kwargs))
+            )
+            data = flatten_context(new_context)
```

Comment:
Any combination, apparently :https://travis-ci.org/divio/django-cms/jobs/104096315 (this is for dj 1.8 / py3.5)
This is the line where tests fail: https://github.com/divio/django-cms/blob/develop/cms/templatetags/cms_tags.py#L1064